extendr-api/src/robj/try_from_robj.rs

```diff
@@ -178,6 +178,30 @@ impl TryFrom<Robj> for Vec<f64> {
     }
 }
 
+impl TryFrom<Robj> for Vec<Bool> {
+    type Error = Error;
+
+    fn try_from(robj: Robj) -> Result<Self> {
+        if let Some(v) = robj.as_logical_slice() {
+            Ok(Vec::from(v))
+        } else {
+            Err(Error::ExpectedInteger(robj))
+        }
+    }
+}
+
+impl TryFrom<Robj> for Vec<u8> {
+    type Error = Error;
+
+    fn try_from(robj: Robj) -> Result<Self> {
+        if let Some(v) = robj.as_raw_slice() {
+            Ok(Vec::from(v))
+        } else {
+            Err(Error::ExpectedInteger(robj))
```

Comment:
Is this the right error? It looks like you're interpreting the input in raw format, shouldn't the error then be something about not being able to use the data in raw format?

Reply:
Done